sdcm/utils/raft/__init__.py

```diff
@@ -92,21 +96,22 @@ def get_group0_non_voters(self) -> list[str]:
         # Add host id which cann't vote after decommission was aborted because it is fast rebooted / terminated")
         return [member['host_id'] for member in self.get_group0_members() if not member['voter']]
 
-    def diff_token_ring_group0_members(self) -> list[str]:
-        LOGGER.debug("Compare token ring and group0 members")
+    def get_diff_group0_token_ring_members(self) -> list[str]:
```

Comment:
looks like this could be renamed to `get_missing_token_ring_members`

Reply:
i would like to preserve the current name, because it say exactly difference between group0 and token ring